### PR TITLE
delete non existent layer normalization

### PR DESCRIPTION
**Pull request by HagegeR**
this just make every node bigger for no reason, batch_normalize is just a property

### CODE REVIEW

**Comment by lutzroeder** [comment]
Most frameworks model batch normalization as a separate layer or an inner layer as it is a separate operation using weight tensors. It currently renders as an inner node for consistency.

**Comment by HagegeR** [comment]
not in Darknet, in Darknet it's just an option that you turn on or off.
